spec/factories/user.rb

```diff
@@ -0,0 +1,9 @@
+FactoryBot.define do
+  factory :user do
+    name { Faker::Name.name }
```

Comment:
Suggestion here, try to align all the brackets here, it will make it easier to read